src/service/webhook

```diff
@@ -1,250 +1,279 @@
+            if not os.path.exists(filename):
+                return
+            data = {}
+            with open(filename) as fd:
+                for line in fd.readlines():
+                    line = line.strip()
+                    if line[0] == "#":
+                        continue
+                    try:
+                        key, value = line.split("=", 1)
+                    except ValueError:
+                        continue
+                    key = key.strip()
+                    value = value.strip()
+                    data[key] = value
+        else:
+            data = os.environ
+
+        defaults = {
+            "WH_USER": "admin",
+            "WH_PASSWD": "root",
+            "OBS": "mer",
+            "WEBHOOK_URL": "http://127.0.0.1:8000/webhook/api",
+            "DEBUG": False,
+        }
+        for key, value in defaults.iteritems():
+            if key in data:
+                value = data[key]
+            setattr(self, key, value)
+
+    def _parse_args(self, args):
+        parser = argparse.ArgumentParser(
+            description=__doc__,
+            formatter_class=argparse.RawTextHelpFormatter,
+        )
+        parser.add_argument(
+            '--repourl',
+            required=True,
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--branch',
+            required=True,
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--token',
+            default='',
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--debian',
+            choices=['Y', 'N'],
+            default='N',
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--dumb',
+            choices=['Y', 'N'],
+            default='N',
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--build',
+            choices=['true', 'false'],
+            default='true',
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--notify',
+            choices=['true', 'false'],
+            default='true',
+            help=argparse.SUPPRESS,
+        )
+        parser.add_argument(
+            '--comment',
+            default='',
+            help=argparse.SUPPRESS,
+        )
+        self.args = parser.parse_args(args=args)
+
+    @property
+    def old_whm(self):
+        if not hasattr(self, '_old_whm'):
+            params = {
+                'obs': self.OBS,
+                'project': self.project,
+                'package': self.package,
+                #'repourl': self.args.repourl,
+                #'branch': self.args.branch,
+            }
+            self._debug('Fetching old whm')
+            self._debug(params)
+            r = requests.get(
+                '%s/webhookmappings/' % self.WEBHOOK_URL,
+                auth=self.auth,
+                params=params,
+            )
```

Comment:
Will this work?
The old url ended in /find to disambiguate
CURL -X GET $WEBHOOK_URL/webhookmappings/$OBS/$OBS_SERVICE_PROJECT/$OBS_SERVICE_PACKAGE/find/